### ISSUE TITLE
Accessibility: Make filter buttons bigger

### ISSUE THREAD

**Issue by spacefront** [contao/contao#6023]
### Description

Buttons are hard to click exactly.

![image](https://user-images.githubusercontent.com/6522818/236666953-9749d1ce-2082-447b-8d43-31d6960e7ba0.png)

**Comment by zonky2** [comment]
As discussed on Contao Camp 2023 / https://github.com/contao-community-alliance/contao-camps/blob/main/2023/camp-2023.md#samstag-06052023

**Comment by do-while** [comment]
Backend Styles:
```
/* upscale backend filter buttons */
.tl_img_submit {
    width: 32px;
    height: 32px;
    text-indent: 32px;
    top: 0;
}
.filter_apply, .filter_reset {
    background-size: 32px 32px;
}

```

**Comment by aschempp** [comment]
Just as a general thought, these are currently the same size as any row icon (edit, delete etc.) right? Maybe the problem is also related to very small spacing between the two icons?

**Comment by do-while** [comment]
It's not like you can't hit the buttons, they're too inconspicuous and the customer won't notice them.
For some time I always enlarge the buttons with the backend CSS (see above) and the customers cope better with it.

**Comment by fritzmg** [comment]
I don't think the size of the button will fix the underlying UX issue. In our experience it is just not immediately apparent that you have to click the weird shape with the ➕ in order to actually apply the filter (yes, I know it's a funnel and used in Excel too - but it's not me that is having this issue 😉).

**Comment by Aybee** [comment]
> Backend Styles:
> 
> ```
> /* upscale backend filter buttons */
> .tl_img_submit {
>     width: 32px;
>     height: 32px;
>     text-indent: 32px;
>     top: 0;
> }
> .filter_apply, .filter_reset {
>     background-size: 32px 32px;
> }
> ```

```css
/* upscale backend filter buttons */
.tl_img_submit {
  width: 32px;
  height: auto;
  aspect-ratio: 1;
  text-indent: 100%;
  top: 0;
  background-size: 100%;
}
```

**Comment by Aybee** [comment]
My filter works immediately. Is perfect for me.
```js
(function($) {
  window.addEvent('domready', function() {
    var init;

    init = function(){
      $$('.tl_filter select, .tl_sorting select, .tl_search select').addEvent('change', function() {
        if(this.get('name') == 'tl_field' && this.getNext('input').get('value').length == 0) {
          return false;
        }
        this.form.submit();
      });
    };

    // on document ready
    init();

    // after ajax manipulation
    window.addEvent('ajax_change', init);

  });
})(document.id);
```

**Comment by ameotoko** [comment]
I use Contao for 14 years and my eyes are still looking for a "Submit" or "Apply" button😅. Because all the filters and search are essentially a big form, and you would naturally expect a submit button to be the most prominent element visually. I always had been using `Enter` key instead, but that is no longer possible after #3660.

**Comment by fritzmg** [comment]
> I always had been using `Enter` key instead, but that is no longer possible after #3660.

Why?

**Comment by ameotoko** [comment]
Beats me🤷🏻‍♂️ probably the same reason why `accesskey` does not work inside TinyMCE instances - the library just does not let them propagate or something.

**Comment by fritzmg** [comment]
No I mean - what is the exact issue. Using `Enter` works fine in the search field.

**Comment by ameotoko** [comment]
Yes, but not from select/chosen fields. That’s not a big deal of course, just an inconvenience. 

Since the Chosen widget itself has a search field built-in, you would also naturally try to submit by pressing Enter at first.

**Comment by fritzmg** [comment]
> Yes, but not from select/chosen fields.

`Enter` has no effect there in Contao 4.9 either. Anyway this is an unrelated discussion. If you think there is a bug/regression, please open a separate issue.

**Comment by aschempp** [comment]
On the original issue: maybe we should just use regular buttons ("Apply" and "Reset") instead of funnel icons?

**Comment by m-vo** [comment]
On June 29th we briefly discussed this issue in the Contao call.

One idea was to give the filters more room, like a new line when a filter is active and where you can disable it again. Here is an example of such a UI from the IKEA website:

![image](https://github.com/contao/contao/assets/5305677/89f0fea1-c666-48ef-8c9e-dd6af3df80fa)

**Comment by zoglo** [comment]
See #7594

**Comment by fritzmg** [comment]
Reopening as the size of the filter buttons did not actually change with #7594

**Comment by zoglo** [comment]
It did increase the size of the buttons but does not include the text, I agree 👍

**Comment by fritzmg** [comment]
@spacefront are the changes in 5.5 sufficient? The size of the buttons were increased from 16x16px to 30x30px

**Comment by leofeyer** [comment]
> Reopening as the size of the filter buttons did not actually change with [#7594](https://github.com/contao/contao/pull/7594)

What is that supposed to mean? The button size has almost doubled in Contao 5.5!

### Contao 5.4

<img width="458" alt="" src="https://github.com/user-attachments/assets/6c89fe59-14a8-44a3-ab33-08e4f1feb459" />

### Contao 5.5

<img width="451" alt="" src="https://github.com/user-attachments/assets/7be15df6-4617-4fd2-8ad2-8f118b654298" />

**Comment by zoglo** [comment]
> [spacefront](https://github.com/spacefront) are the changes in 5.5 sufficient? The size of the buttons were increased from 16x16px to 30x30px

[WCAG - Target size minimum](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html)
At least 24 x 24 px is needed to pass `AA`

[WCAG - Target size](https://www.w3.org/WAI/WCAG21/Understanding/target-size.html)
`AAA` is not mandatory / not needed right now and would need at least 44 x 44 px

**Comment by fritzmg** [comment]
Ah yes, then we can close this after all (only noticed that the referenced PR originally had a larger button - i.e. with text, which was then ultimately not merged).
